configure.ac

```diff
@@ -1239,8 +1239,8 @@ IPv6_CHECK_FUNC(getaddrinfo, [IPv6_CHECK_FUNC(gai_strerror,
                 sasl_cv_getaddrinfo=no])])
 if test $sasl_cv_getaddrinfo = yes; then
     AC_LIBOBJ(getaddrinfo)
-    GETADDRINFOOBJS="getaddrinfo.o"
-    LTGETADDRINFOOBJS="getaddrinfo.lo"
+    GETADDRINFOOBJS="../lib/getaddrinfo.o"
```

Comment:
Does this avoid the need for f9f28ac09bf35336eb04782258bec8dd5c6767b8 to exist (or make it a deletion)?

Reply:
I believe those other files ought to be deleted, yeah. But the purpose of this PR was mainly to import old patches we were already carrying. Other cleanup can be a separate PR.

Reply:
Deleted redundant files